src/Symfony/Bridge/Monolog/Formatter/ConsoleFormatter.php

```diff
@@ -11,45 +11,186 @@
 
 namespace Symfony\Bridge\Monolog\Formatter;
 
-use Monolog\Formatter\LineFormatter;
-use Monolog\Logger;
+use Monolog\Formatter\FormatterInterface;
+use Symfony\Component\VarDumper\Cloner\Data;
+use Symfony\Component\VarDumper\Cloner\Stub;
+use Symfony\Component\VarDumper\Cloner\VarCloner;
+use Symfony\Component\VarDumper\Dumper\CliDumper;
 
 /**
  * Formats incoming records for console output by coloring them depending on log level.
  *
  * @author Tobias Schultze <http://tobion.de>
+ * @author Grégoire Pineau <lyrixx@lyrixx.info>
  */
-class ConsoleFormatter extends LineFormatter
+class ConsoleFormatter implements FormatterInterface
 {
-    const SIMPLE_FORMAT = "%start_tag%[%datetime%] %channel%.%level_name%:%end_tag% %message% %context% %extra%\n";
+    const SIMPLE_FORMAT = "%datetime% %level_start%%level_name%%level_end% [%channel%] %message%%context%%extra%\n";
+    const SIMPLE_DATE = 'H:i:s';
+
+    private static $levelColorMap = array(
+        100 => 'fg=white',
+        200 => 'fg=green',
+        250 => 'fg=blue',
+        300 => 'fg=cyan',
+        400 => 'fg=yellow',
+        500 => 'fg=red',
+        550 => 'fg=red',
+        600 => 'fg=white;bg=red',
+    );
+
+    private $options;
+    private $cloner;
+    private $outputBuffer;
+    private $dumper;
 
     /**
      * {@inheritdoc}
      */
-    public function __construct($format = null, $dateFormat = null, $allowInlineLineBreaks = false, $ignoreEmptyContextAndExtra = true)
+    public function __construct($options = array())
     {
-        parent::__construct($format, $dateFormat, $allowInlineLineBreaks, $ignoreEmptyContextAndExtra);
+        // BC Layer
+        if (!is_array($options)) {
+            $args = func_get_args();
+            $options = array();
+            if (isset($args[0])) {
+                $options['format'] = $args[0];
+            }
+            if (isset($args[1])) {
+                $options['date_format'] = $args[1];
+            }
+        }
+
+        $this->options = array_replace(array(
+            'format' => self::SIMPLE_FORMAT,
+            'date_format' => self::SIMPLE_DATE,
+            'colors' => true,
+            'multiline' => false,
+        ), $options);
+
+        $this->cloner = new VarCloner();
```

Comment:
No indeed. IMHO it's useless and there are other part of the framework where the dumper / cloner is hard-coded like that.

About the line 83, I cannot anyway as I do need a CLIDumper and well configured.